plugins/mcreator-localization/lang/texts.properties

```
@@ -3097,7 +3097,7 @@ validators.registry_name.invalid={0} can only contain lowercase English letters,
 validators.registry_name.invalid2={0} can''t end with a colon character
 validators.registry_name.invalid3={0} can''t contain more than one colon character
 validators.unique_name.empty={0} can''t be empty
-validators.unique_name.duplicate=This name is already used
+validators.unique_name.duplicate=The specified name {0} is already used
```

I don't think that the _name_ word should be here (imagine `Property name` as the argument, would `The specified name property name is already used` sound normal?).

---

In fact I guess `The specified ` is not needed then too:
```suggestion
validators.unique_name.duplicate={0} is already used
```

---

Then, we should maybe use quotes? Like this : `The specified name "property name" is already used`

---

The other string does not, as well as strings used by all other validators.

---

So should there be quotes on all validator strings?

---

Check my latest commit

---

At least for the specified name wording,

`{0} is already used`

works with or without probably, but Max already went with " so all good